backend/src/main/resources/schema.graphqls

```diff
@@ -222,6 +224,24 @@ type Mutation {
     deviceTypes: [String]!
     defaultDevice: String!
   ): Organization
+  addUser(
+    firstName: String
+    middleName: String
+    lastName: String
+    suffix: String
+    email: String
+    organizationExternalId: String
+  ): ID
```

Comment:
I'm informed (by Dave) that graphQL best practice is to return the object that was created or updated. Of course I'm not entirely sure what we're creating here (I'm open to arguments as to whether we create the DB object or not), but once we figure that out I would prefer that we return the object.

Reply:
looping in @dmethvin-gov too -- i decided to return ID based on the fact that the patient queries return an ID. I'm happy to return the object here, though that'll make our return patterns inconsistent. is that okay?

Reply:
It is definitely OK—let's get confirmation from Dave on what we should actually be returning, though.

Reply:
Dave says whole object, I'll make the fix!